src/evaluatorfrontend.cpp

```diff
@@ -67,18 +74,82 @@ void EvaluatorFrontend::FuncRunner() {
 		Plot();
 }
 
-//! @file
-// [runeval]
-void EvaluatorFrontend::RunEvaluator() {
-	while (!evaluator->IsFinished()) {
+int EvaluatorFrontend::EvaluatorFunc() {
+	if (!evaluator->IsFinished()) {
 		states.push_back(evaluator->GetNextNetworkState());
 		if (printStd) {
 			std::cout << states.back().PrintCsvRow();
 		}
+	} else {
+		return 0;
 	}
+
+	return 1;
+}
+
+//! @file
+// [runeval]
+void EvaluatorFrontend::RunEvaluator() {
+	while (EvaluatorFunc())
+		;
 }
 // [runeval]
 
+void EvaluatorFrontend::RunRTEvaluator() {
+	auto StartT = std::chrono::steady_clock::now();
+	auto EndT = std::chrono::steady_clock::now();
+	bool once = true;
+
+	while (!mPlot->RunPlot()) {
+		if (EvaluatorFunc()) {
+			EndT = std::chrono::steady_clock::now();
+			auto ETime =
```

Comment:
Maybe write out the variable names. `elapsedTime`, or whatever they are meant to mean.